src/chain/response.rs

```diff
@@ -405,3 +405,56 @@ impl TryFrom<EventAttribute> for Tag {
         })
     }
 }
+
+impl From<tendermint_rpc::endpoint::abci_query::AbciQuery> for ChainResponse {
+    fn from(res: tendermint_rpc::endpoint::abci_query::AbciQuery) -> ChainResponse {
+        ChainResponse {
+            code: todo!(), // res.code.into(),
+            data: Some(res.value),
+            log: res.log.to_string(),
+        }
+    }
+}
+
+impl From<tendermint_rpc::endpoint::broadcast::tx_async::Response> for AsyncChainTxResponse {
+    fn from(res: tendermint_rpc::endpoint::broadcast::tx_async::Response) -> Self {
+        Self {
+            res: ChainResponse {
+                code: todo!(), // res.code.into(),
+                data: Some(res.data.into()),
+                log: res.log.to_string(),
+            },
+            tx_hash: res.hash.to_string(),
+        }
+    }
+}
+
+impl From<tendermint_rpc::endpoint::broadcast::tx_sync::Response> for AsyncChainTxResponse {
+    fn from(res: tendermint_rpc::endpoint::broadcast::tx_sync::Response) -> Self {
+        Self {
+            res: ChainResponse {
+                code: todo!(), // res.code.into(),
+                data: Some(res.data.into()),
+                log: res.log.to_string(),
+            },
+            tx_hash: res.hash.to_string(),
+        }
+    }
+}
+
+impl From<tendermint_rpc::Code> for Code {
+    fn from(value: tendermint_rpc::Code) -> Code {
+        match value {
+            tendermint_rpc::Code::HttpError => todo!(),
+            tendermint_rpc::Code::WebSocketError => todo!(),
+            tendermint_rpc::Code::ClientInternalError => todo!(),
+            tendermint_rpc::Code::ParseError => todo!(),
+            tendermint_rpc::Code::InvalidRequest => todo!(),
+            tendermint_rpc::Code::MethodNotFound => todo!(),
+            tendermint_rpc::Code::InvalidParams => todo!(),
+            tendermint_rpc::Code::InternalError => todo!(),
+            tendermint_rpc::Code::ServerError => todo!(),
+            tendermint_rpc::Code::Other(_) => todo!(),
+        }
+    }
+}
```

Comment:
If we changed the `tendermint-rpc` dep to `0.26`, then we wouldn't have to re-implement these impls since we already have them defined for the same re-exported types coming from cosmrs. 

I know it sucks to use an older version but we can upgrade in another commit in lockstep with cosmrs (thanks for making that issue btw).